### DISCUSSION TITLE
No useOutletContext?

### DISCUSSION THREAD

**Discussion by justinwyllie** [remix-run/react-router#8505] [Q&A]
On this page in the docs: https://reactrouter.com/docs/en/v6/api#useoutletcontext there are come examples for useOutletContext. (To pass data from a parent component to one rendered by Outlet )

But useOutletContext is not available for export in react-router-dom 6.0.2.

Maybe I need to upgrade? 

Thank you

**Comment by jovand3v** [comment]
I've had the same problem, fixed it by upgrading react-router-dom from 6.0.2 to 6.2.1, also make sure you restart the server once you update the dependency.
